samples/matter/common/src/binding_handler.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#pragma once
+
+#include <app-common/zap-generated/ids/Clusters.h>
+#include <app-common/zap-generated/ids/Commands.h>
+#include <app/CommandSender.h>
+#include <app/clusters/bindings/BindingManager.h>
+#include <controller/InvokeInteraction.h>
+#include <platform/CHIPDeviceLayer.h>
+
+class BindingHandler {
+public:
```

Comment:
What I miss here is the documentation of the public API. We can however deffer that to followup task/PR.

Reply:
Also, isn't the BindingHandler stateless class? Look like we are using class only to provide the meaningful namespace. The same can be achieved with just `namespace` and regular functions. So object oriented code makes sense to me if we foresee possible extensions of the class in the future. I don't have a string opinion on that, and I accept the contribution. It just looks a bit odd to have a class with all members static :)